src/components/ga4/EventBuilder/ValidateEvent/schemas/baseContent.ts

```diff
@@ -0,0 +1,28 @@
+// Base JSON Body Content Schema.
+
+// from google3.corp.gtech.ads.infrastructure.mapps_s2s_event_validator.schemas import events
+import { userPropertiesSchema } from './userProperties'
+import { eventsSchema } from './events'
+
+export const baseContentSchema = {
+    "type": "object",
+    "required": ["app_instance_id", "events"],
+    "additionalProperties": false,
+    "properties": {
+        "app_instance_id": {
+            "type": "string",
+            "format": "app_instance_id"
+        },
+        "user_id": {
+            "type": "string"
+        },
+        "timestamp_micros": {
+            // "type": "number"
```

Comment:
Will this work with the type commented out?

Reply:
No, this won't validate timestamp_micros to be a number. We need to update the input field for timestamp_micros from string to number in order to use this validation